templates/foundation.html

```diff
@@ -72,7 +72,7 @@ <h2>Leadership</h2>
             <div>Secretary</div>
         </div>
     </div>
-    
+
```

Comment:
These white space changes seem to be inline with the rest of the file

Reply:
could you remove those extra changes? they are probably introduced by your IDE auto formatter

Reply:
```suggestion
    
```